src/main/java/com/regnosys/testing/testpack/TestPackFunctionRunnerImpl.java

```diff
@@ -75,10 +75,11 @@ public TestPackFunctionRunnerImpl(Function<IN, RosettaModelObject> function,
 
     @Override
     public Pair<String, Assertions> run(Path inputPath) {
-        URL inputFileUrl = Resources.getResource(inputPath.toString());
-        IN input = readFile(inputFileUrl, JSON_OBJECT_MAPPER, inputType);
         RosettaModelObject output;
         try {
+            Path inputPathFromRepositoryRoot = ROSETTA_SOURCE_PATH.resolve(inputPath);
+            URL inputFileUrl = inputPathFromRepositoryRoot.toUri().toURL();
+            IN input = readFile(inputFileUrl, JSON_OBJECT_MAPPER, inputType);
             output = function.apply(resolveReferences(input));
         } catch (Exception e) {
```

Comment:
```
} catch (MalformedUrlException e) {
    LOGGER.error("Failed to load input path {}", inputPath, e);
    return Pair.of(null, new Assertions(null, null, true));
} catch (Exception e) {
```

Reply:
So you can tell that the exception was caused by path, rather than the running of the function